src/components/sections/cta/cta.types.ts

```diff
@@ -14,7 +15,7 @@ export type bgVariant =
   | 'rigth-bottom-to-top-left'
   | 'left-top-to-bottom-right';
 
-export interface CTAProps extends Component {
+export interface CTAProps extends Component, Pick<DownloadButtonProps, 'fixedOS'> {
   variant?: CTAVariant;
```

Comment:
The prop `fixedOS` in the context of the CTA does not have any meaning. Could we rename it to a more descriptive name purpose? Something like `downloadButtonDefaultOsValue`?

```suggestion
export interface CTAProps extends Component {
  downloadButtonDefaultOsValue?: DownloadButtonProps['fixedOS'];
  variant?: CTAVariant;
```